glossary.yml

```diff
@@ -392,6 +398,13 @@
     def: "በ [ነገር-ተኮር መርሃግብር](#oop) ውስጥ አንድ [ዘዴ](#method) ተገልፇል ነገር ግን አልተተገበረም ፡፡
       መርሃግብሮች [ወላጅ] ውስጥ ረቂቅ ዘዴን ይገልፃሉ ክፍል](#parent_class) [የህፃናት ክፍሎች](#child_class)
       ማቅረብ አለበት ፡፡ \n"
+
+  fr:
+    term: "méthode abstraite"
+    def: >
+      En programmation orientée objet, c'est la méthode définie mais non implémentée. 
```

Comment:
Suggest changing to 

En programmation orientée objet, une méthode définie mais non implémentée. Les programmeurs définissent une méthode abstraite dans une classe parente pour spécifier les opérations que les classes enfants doivent fournir.